auth-web/src/components/auth/common/ModalDialog.vue

```diff
@@ -94,4 +94,9 @@ export default class ModalDialog extends Vue {
     justify-content: center;
     padding: 1.5rem;
   }
+  // Info Dialog Variant
+ .info-dialog .v-card__actions {
+    justify-content: center;
+    padding: 1.5rem;
+  }
```

Comment:
Please show a screenshot with the effect of this change.

Reply:
It's to add a new css style for dialog. It's used for unaffiliated modals only (refers to uxpin https://preview.uxpin.com/a86dccdd20ac62828f965c5eea10f81b828a9853#/pages/163299967/simulate/sitemap).

Reply:
OK. Can you show a screenshot of it from your branch?

Reply:
Same as the below comments 😃 It keeps action button centered while the title and texts align left.
![image](https://github.com/bcgov/sbc-auth/assets/60866283/a98fa98d-2afd-423c-a0a9-5bd2707af33e)